src/character/class/mod.rs

```diff
@@ -21,4 +24,58 @@ impl From<&str> for ClassId {
 }
 
 /// Describes all bonuses and maluses of a class.
-pub trait ClassModel {}
+pub trait ClassModel {
+    /// Returns the hit dice per level.
+    fn hit_dice(&self) -> HitDice;
+
+    /// Returns the base hit points at first level.
+    fn hit_points_at_1st_level(&self) -> u8 {
+        self.hit_dice().die.max()
+    }
+
+    /// Returns the proficiency bonus at the given level.
+    fn proficiency_bonus(&self, level: Level) -> ProficiencyBonus {
+        proficiency_bonus_at_level(level)
+    }
+}
+
+/// Returns the standard proficiency bonus at the given level.
+fn proficiency_bonus_at_level(level: Level) -> ProficiencyBonus {
+    ((level.value() - 1) / 4 + 2).into()
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn standard_proficiency_bonus() {
+        macro_rules! test_case {
+            ($( $lvl:expr, $expected:expr ),* $(,)?) => {
+                $(
+                    assert_eq!(proficiency_bonus_at_level(Level::new($lvl).unwrap()), $expected.into());
+                )*
+            }
+        }
+        test_case!(1, 2);
```

Comment:
perhaps we could wrap the `macro` into a `macro` with ranges .. like `test_cases!(1..=4, 2)`
I have no idea how to do that